core/src/main/java/com/amazon/opendistroforelasticsearch/sql/planner/physical/FilterOperator.java

```diff
@@ -0,0 +1,55 @@
+package com.amazon.opendistroforelasticsearch.sql.planner.physical;
```

Comment:
Could you add license for this source file?

Reply:
done.